src/args.rs

```diff
@@ -1,144 +1,63 @@
-use clap::{App, AppSettings, Arg};
+extern crate docopt;
+
+use docopt::Docopt;
 use std::path::PathBuf;
-use std::time::Duration;
 use rusoto_core::Region;
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Deserialize, Clone)]
 pub struct Args {
-    pub gerrit_hostname: String,
-    pub gerrit_port: u16,
-    pub gerrit_username: String,
-    pub gerrit_priv_key_path: PathBuf,
-    pub spark_url: String,
-    pub spark_endpoint: String,
-    pub spark_sqs: String,
-    pub spark_sqs_region: Region,
-    pub spark_webhook_url: Option<String>,
-    pub spark_bot_token: String,
-    pub verbosity: usize,
-    pub quiet: bool,
-    pub bot_msg_expiration: Duration,
-    pub bot_msg_capacity: usize,
+    pub flag_gerrit_hostname: String,
+    pub flag_gerrit_port: u16,
+    pub flag_gerrit_username: String,
+    pub flag_gerrit_priv_key_path: PathBuf,
+    pub flag_spark_url: String,
+    pub flag_spark_endpoint: String,
+    pub flag_spark_sqs: String,
+    pub flag_spark_sqs_region: Region,
+    pub flag_spark_webhook_url: Option<String>,
+    pub flag_spark_bot_token: String,
+    pub flag_verbose: bool,
+    pub flag_quiet: bool,
+    pub flag_bot_msg_expiration: u64,
+    pub flag_bot_msg_capacity: usize,
 }
 
-const SPARK_URL: &str = "https://api.ciscospark.com/v1";
+const USAGE: &'static str = "
+Cisco Spark <> Gerrit Bot
 
-const USAGE: &str = r#"
--v, --verbose... 'Verbosity level'
--q, --quiet      'Quiet'
-"#;
+Usage:
+  gerritbot-rs [-v] --gerrit-hostname=<HOSTNAME> --gerrit-port=<PORT> --gerrit-username=<USERNAME> --gerrit-priv-key-path=<FILE> [--spark-url URL] [--spark-endpoint URL] [--spark-sqs URI] [--spark-sqs-region REGION] [--spark-webhook URL] [--spark-bot-token TOKEN] [--bot-msg-expiration SECS] [--bot-msg-capacity CAPACITY] 
```

Comment:
I think you can just write `[OPTIONS]`

Reply:
Pushed new commit on top. It doesn't work with sections though, so I removed those.